src/App.vue

```diff
@@ -619,6 +651,18 @@ export default {
 				this.$router.push({ name: 'root' })
 			}
 		},
+
+		/**
+		 * Emits the UNREAD_COUNT_UPDATED event with the current counter values
+		 */
+		emitUnreadCountUpdated() {
+			emit('talk:unread:updated', {
```

Comment:
As per the comment above
```suggestion
		emitUnreadCountUpdated() {
			emit('talk:unreads:updated', {
```